exercises/concept/vehicle-purchase/.docs/hints.md

```diff
@@ -0,0 +1,20 @@
+# Hints
+
+## General
+
+- You can accomplish all of these tasks without using an `if` statement.
+
+## 1. Determine if you will need a drivers license
+
+- Use the [equality operator][cmp] to check whether the first field equals a certain string.
+- Use a a regular expression or the `index` function to check if the second field contains the required substrings.
+- Use one of the two [boolean operators] to combine the two requirements.
```

Comment:
Does this link not need a source when the source matches the text?

Reply:
https://spec.commonmark.org/0.31.2/#shortcut-reference-link

> A [shortcut reference link](https://spec.commonmark.org/0.31.2/#shortcut-reference-link) consists of a [link label](https://spec.commonmark.org/0.31.2/#link-label) that [matches](https://spec.commonmark.org/0.31.2/#matches) a [link reference definition](https://spec.commonmark.org/0.31.2/#link-reference-definition) elsewhere in the document and is not followed by `[]` or a link label. The contents of the link label are parsed as inlines, which are used as the link’s text. The link’s URI and title are provided by the matching link reference definition. Thus, `[foo]` is equivalent to `[foo][]`.